src/preamble.js

```diff
@@ -978,12 +978,38 @@ function createWasm(env) {
     return value;
   };
 #endif
+var proxyHandler = {
+  'get': function(obj, prop) {
+    if (prop in obj) {
+      return obj[prop]; // already present
+    }
+    
+    if (prop.startsWith('invoke_')) {
+      // A missing invoke, i.e., an invoke for a function type
+      // present in the dynamic library but not in the main JS,
+      // and the dynamic library cannot provide JS for it. Use
+      // the generic "X" invoke for it.
+      return obj[prop] = invoke_X;
+    }
+
+    if (Module[prop]) {
+      return Module[prop];
+    }
+
+    // if not a global, then a function - call it indirectly
```

Comment:
This comment seems wrong.

Seems like it should say something like:  "For a missing property generate a stub that will do a runtime lookup and error out if its still missing".

Reply:
Sure, will update it.